packages/react-core/src/components/Select/SelectOption.tsx

```diff
@@ -127,6 +170,7 @@ export class SelectOption extends React.Component<SelectOptionProps> {
                     isDisabled && styles.modifiers.disabled,
                     isFocused && styles.modifiers.focus,
                     description && styles.modifiers.description,
+                    isFavorite && styles.modifiers.link,
```

Comment:
It looks like the `pf-m-link` modifier is only applying to the `pf-c-select__menu-item` when it is a favorited item. The `pf-m-link` modifier should apply to the `pf-c-select__menu-item` that holds the main item text for the list item any time there are multiple actions in an item. `pf-m-link` is what differentiates the main item text `pf-c-select__menu-item` from other action `pf-c-select__menu-item`s.

Another way of saying that is that any time the parent is `pf-c-select__menu-wrapper`, the `pf-c-select__menu-item` that holds the item text needs to have `pf-m-favorite`. Re-using the test above, it seems like this could be achieved via

```suggestion
                    isFavorite !== null && styles.modifiers.link,
```